manifests/m/MullvadVPN/MullvadVPN/2024.8/MullvadVPN.MullvadVPN.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: MullvadVPN.MullvadVPN
+PackageVersion: "2024.8"
+AppsAndFeaturesEntries:
+- DisplayVersion: "2024.8.0"
```

Comment:
```suggestion
```
`DisplayVersion` isn't necessary here, as the package manager adds trailing zeroes.

Reply:
Ah, I see! So that's why it wasn't necessary here, where it was in `MullvadVPN.MullvadVPN.beta`. Thanks for pointing it out